src/redux/actions/basketAction.js

```diff
@@ -30,24 +41,42 @@ function* addProductToWatcher(){
     yield takeEvery(ADD_PRODUCT_TO_BASKET, addProductToBasketWorker)
 }
 
-
 function* fetchBasketWorker(){
-    const data = yield call(ApiService.load, "dataBasket");
+    const basketData = yield call(ApiService.load, "dataBasket");
+    // console.log(basketData);
```

Comment:
Delete `// console.log(basketData);`.
And add `try {} catch(err) {}` to every function where you use backend request